charts/azure-managed-cluster/README.md

```diff
@@ -0,0 +1,154 @@
+# CAPI Azure
+
+## Prerequisites
+- [Docker Desktop](https://www.docker.com/)
+- [Kind](https://kind.sigs.k8s.io/)
+- [ClusterCTL](https://cluster-api.sigs.k8s.io/clusterctl/overview.html) Version v1.6.1 or older
+- [Helm](https://helm.sh) version v3.14.0 or later
+
+## Prerequisites Installations
+- Docker Desktop
+ https://www.docker.com/products/docker-desktop 
+
+- Install Kind
+  https://kind.sigs.k8s.io/
+
+- Install Clusterctl
+  https://cluster-api.sigs.k8s.io/clusterctl/overview.html
+
+- Install Helm3
+- https://helm.sh/docs/intro/install/
+
+## Deploy
+
+Clone this repo
+
+```bash
+git clone https://github.com/ams0/azure-managed-cluster-capz-helm.git
```

Comment:
These steps should use the `helm repo add capz https://kubernetes-sigs.github.io/cluster-api-provider-azure` experience, not cloning the repo

Reply:
@dtzar  should i update the chart name to `cluster-api-provider-azure` instead of  `azure-managed-cluster-capz-helm` otherwise it would be 

`helm repo add capz https://kubernetes-sigs.github.io/azure-managed-cluster-capz-helm`